app/views/proposals/details/_summary.html.haml

```diff
@@ -17,8 +14,16 @@
             %strong
               %span
                 = proposal.created_at.strftime("%b %d, %Y at %l:%M%P")
-      .medium-3.column.text-center
-        %span.status-wrap.medium-12.small-12{ class: "status-is-" + @proposal.status }
+        %h1.c2n_header
+          = proposal.name
+      .medium-3.column.text-right
+        %span.status-wrap.medium-12.small-12{ class: "status-is-" + proposal.status }
           Status:
           %span.status-value
-            = @proposal.capitalized_detailed_status
+            = proposal.capitalized_detailed_status
+          %span.status-icon
+        - if proposal.status == "completed"
+          %span.completed-time
+            on 
+            = proposal.steps.last.updated_at.strftime("%b %d, %Y at %l:%M%P")
```

Comment:
Do you want `.completed_at` instead? That date is more "official" than the `updated_at` date, esp if the step is re-started.